src/redisraft.c

```diff
@@ -2171,10 +2171,11 @@ __attribute__((__unused__)) int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisMod
                     REDISRAFT_VERSION, REDISRAFT_GIT_SHA1);
 
     const int MIN_SUPPORTED_REDIS_VERSION = 0x00070000;
```

Comment:
We can update `MIN_SUPPORTED_REDIS_VERSION` too - to `0x00070200`

Reply:
7.2 is still RC. Looks like we don't update version numbers in RCs. See [link](https://github.com/redis/redis/blob/7.2/src/version.h)
It is still 7 bla bla. 

Maybe we should ignore this for now?

Reply:
k, good to know, in that case - sure, we can update later (just open a ticket so we will do it in the future, thanks)